helm/prometheus-rules/templates/alerting-rules/cert-manager.rules.yml

```diff
@@ -39,3 +39,15 @@ spec:
         severity: page
         team: cabbage
         topic: cert-manager
+    - alert: CertManagerTooManyCertificateRequests
+      annotations:
+        description: '{{`There are too many CertificateRequests in cluster {{ $labels.cluster_id }}.`}}'
+        opsrecipe: cert-requests-too-many/
+      expr: sum by (cluster_id) (etcd_kubernetes_resources_count{kind="certificaterequests.cert-manager.io"}) > 5000
+      for: 15m
+      labels:
+        area: managedapps
```

Comment:
is managedapps ok?

Reply:
`nginx-ingress-controller` and `external-dns` - for which Cabbage is also responsible for - uses `managedservices`.

Reply:
https://github.com/giantswarm/prometheus-rules/blob/master/helm/prometheus-rules/templates/alerting-rules/external-dns.rules.yml
https://github.com/giantswarm/prometheus-rules/blob/master/helm/prometheus-rules/templates/alerting-rules/ingress-controller.rules.yml

Reply:
```suggestion
        area: managedservices
```